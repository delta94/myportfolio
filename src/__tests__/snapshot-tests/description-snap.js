import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import Description from '../../components/description';

it('Description renders correctly', () => {
  const wrapper = shallow(<Description color="#000" text="This is a test description" />);
  expect(toJSON(wrapper)).toMatchSnapshot();
});
