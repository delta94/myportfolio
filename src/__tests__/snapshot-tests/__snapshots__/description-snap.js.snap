// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Description renders correctly 1`] = `
<styled.div>
  <styled.span
    color="#000"
  >
    This is a test description
  </styled.span>
</styled.div>
`;
